File: day-00/code.c
#include <stdio.h>
#include <string.h>

int main() {
    char name[20];

    printf("Name: ");
    scanf("%[^\n]s", name); // Anish, Anish Shobith

    if (strlen(name))
        printf("Hello, %s!", name);
    else
        printf("Hello, SJEC!");
}

File: day-00/Code.java
import java.util.Scanner;

public class Code {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Name : ");
        String name = sc.nextLine();
        if (name.length() > 0) 
            System.out.printf("Hello, %s!", name);
        else
            System.out.print("Hello, SJEC!");
    }
}